x-pack/plugins/task_manager/server/plugin.ts

```diff
@@ -179,7 +179,7 @@ export class TaskManagerPlugin
     core.status.set(
       combineLatest([core.status.derivedStatus$, serviceStatus$]).pipe(
         map(([derivedStatus, serviceStatus]) =>
-          serviceStatus.level > derivedStatus.level ? serviceStatus : derivedStatus
+          serviceStatus.level >= derivedStatus.level ? serviceStatus : derivedStatus
```

Comment:
This way, if `taskManager` status is the same level as core services, the taskManager message will be reported.